src/iris/api.py

```diff
@@ -678,7 +696,8 @@ def stream_incidents_with_context(results, title=False):
                 row['title'] = row['context'].get(title_variable_name)
             else:
                 row['title'] = None
-        yield row
+        results_list.append(row)
+    return results_list
```

Comment:
With this change to this function, from being a generator yielding rows as it decodes them and instead accumulating them all at once in a list and return, it is no longer "streaming" the results and the name "stream_incidents_with_context" becomes misleading.

Can we either revert this change or rename the function?